src/main/java/com/autotune/common/k8sObjects/KruizeObject.java

```diff
@@ -54,14 +54,18 @@ public final class KruizeObject {
     private ExperimentUseCaseType experimentUseCaseType;
     private Set<ExperimentResultData> resultData;
     private ValidationResultData validationData;
+    @SerializedName("cluster_name")
+    private String clusterName;
 
     public KruizeObject(String experimentName,
                         String namespace,
                         String mode,
                         String targetCluster,
                         SloInfo sloInfo,
                         SelectorInfo selectorInfo,
-                        ObjectReference objectReference) throws InvalidValueException {
+                        ObjectReference objectReference,
+                        String clusterNameContent
```

Comment:
Move this parameter to immediately after `experimentName` and it should be called `clusterName`

Reply:
Done.